AnkiDroid/src/main/java/com/ichi2/anki/Reviewer.java

```diff
@@ -61,6 +61,9 @@
     private boolean mBlackWhiteboard = true;
     private boolean mPrefFullscreenReview = false;
     private static final int ADD_NOTE = 12;
+    // Deck picker reset scheduler before opening the reviewer. So
+    // first reset is useless.
+    private boolean reset_done = false;
```

Comment:
If we do this, it's a member variable so it should be mSomething
```suggestion
    private boolean mSchedResetDone = false;
```

Reply:
Sorry. Tim just told me a few day ago about m variables. I really need to get my head around the fact that my PR to anki and akindroid needs to follow different rules. That's extremely unintuitive to be honest, since in my headspace, anki and ankidroid are really related